assets/css/_ui_kit.scss

```diff
@@ -116,6 +116,22 @@ $color-vehicle-ghost-border: $color-component-medium;
 $color-vehicle-nonrevenue: #8f7ed6;
 $color-vehicle-off-course: $color-component-dark;
 
+:root {
+  --color-vehicle-off-course: #{$color-vehicle-off-course};
+  --color-vehicle-ontime: #{$color-vehicle-ontime};
+}
+
+:root,
```

Comment:
**question:** Love this change! One quick thing: why does this get applied to `:root`? Just to make it the default if nothing else is selected?

Reply:
I used `:root: based on observed common CSS practice, and that `:root` makes it available to the entire application. 
I considered adding it to the skate `<main>` tag, but I felt that it would be confusing if this "global" style behaviour changed based on where it was placed (we don't normally place things outside of `<main>` but just in case).

Also `:root` I believe is an alias for `html`, but is more clear what you're targeting in my opinion.

Reply:
also, not something I did, but we should be able to move the `early-red`/`early-blue` styles into the `<html>` tag and then the styles will be application wide, I got lucky in not needing to refactor a bunch of code because we specify those classes at all points needed, but we should be able to move to using only the variables or just the `early`/`late` classes which pull the variables in themselves.

Reply:
Okay, makes sense. And agreed that `:root` is clearer for this purpose than `html`.